confidence.py

```diff
@@ -81,6 +82,12 @@ def _split_keys(mapping, separator='.'):
             # use rest as the new key of value, recursively split that and update value
             value = _split_keys({rest: value}, separator)
 
+        if key in _COLLIDING_KEYS:
+            # warn about configured keys colliding with Configuration members
+            warnings.warn('key {key} collides with member of Configuration type, use get() method to retrieve key '
```

Comment:
This warning is not entirely clear on face value. Use get() method to retrieve the value for {key} would be clearer, I think.

Reply:
Good idea, fixed per 8f5faf0.